### PR TITLE
#139 Fixed an issue where emptying the component would make it unselectable

### PR DESCRIPTION
**Pull request by giorgosart**

### CODE REVIEW

File: src/lib/EasyEdit.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default class EasyEdit extends React.Component {
   }
 
   isNullish(value) {
```
Comment (findhamid) [inline src/lib/EasyEdit.jsx:34]: A good name for a function that checks for null, undefined, or an empty string could be something like isNullOrUndefinedOrEmpty to make its purpose explicit. 

If you prefer an even shorter method name, you can use something like isNUE as an abbreviation for "isNull, Undefined, or Empty." However, remember that shorter names may sacrifice some readability and clarity.

Comment (giorgosart) [reply src/lib/EasyEdit.jsx]: done

File: src/lib/EasyEdit.jsx [hunk 2]
```diff
@@ -413,7 +413,7 @@ export default class EasyEdit extends React.Component {
   renderComplexView() {
     const { placeholder, options, type } = this.props;
 
-    if (this.isNullish(this.state.value) || this.state.value.length === 0) {
+    if (this.isNullOrUndefinedOrEmpty(this.state.value) || this.state.value.length === 0) {
```
Comment (findhamid) [inline src/lib/EasyEdit.jsx:416]: The additional check this.state.value.length === 0 is redundant. The isNullOrUndefinedOrEmpty function already checks if the value is null, undefined, or an empty string, so there is no need to check the length separately.

Comment (giorgosart) [reply src/lib/EasyEdit.jsx]: done

**Review by findhamid — APPROVED** [review]

**Pull request merged**